tests/app/conftest.py

```diff
@@ -1246,3 +1246,24 @@ def app_statsd(mocker):
 
 def datetime_in_past(days=0, seconds=0):
     return datetime.now(tz=pytz.utc) - timedelta(days=days, seconds=seconds)
+
+
+def document_download_response(override={}):
```

Comment:
It was in `tests/app/v2/notifications/test_post_notifications.py` before. Moved it here because it's now used in 2 places